CHANGES/5526.feature

```diff
@@ -0,0 +1 @@
+Add/remove RPMs when a repo's modulemd changes.
```

Comment:
nitpick- can we re-phrase this? it sounds misleading because modulemd is immutable and it does not change.
what about - Add/remove RPMs when a repo's modulemd gets added/removed?